components/core/routes/Acl.py

```diff
@@ -0,0 +1,47 @@
+from flask import json
+from flask import request, abort, Response, g
+from utils.token_utils import token_validator
+from managers.AclManager import *
+
+
+def check_access(id, user_id):
+    token = token_validator(request.headers.get('token'))
+    if token is None:
+        return '{"error":"token error"}', 403
+    try:
+        entity = request.args.get('type')
+        if not entity == 'fr' and not entity =='fl':
+            return Response('{"error":"invalid parameter"}', status=400)
+
+        check_response = get_access(id, user_id, entity)
+        if not isinstance(check_response, str):
```

Comment:
We can probably replace this with an exception. Raise an exception for an error instead of returning a string.

Reply:
will do